src/data/integrations.js

```diff
@@ -283,6 +304,21 @@ const integrations = [
       description: "Athena & Livebook",
     },
   },
+  {
+    id: "snowflake",
+    name: "Snowflake",
+    logo: snowflakeLogo,
+    category: "Data Warehouse",
+    headline: "Connect to Snowflake directly from Livebook",
+    description:
+      "Connect to your Snowflake using Livebook's Database Connection Smart cell. Use Livebook Secrets to store your AWS access key securely. Write and run queries directly from Livebook.",
```

Comment:
We should say somewhere something like "Data exchange happens via the high-performance Apache Arrow format". You may also link to this article: https://www.snowflake.com/blog/fetching-query-results-from-snowflake-just-got-a-lot-faster-with-apache-arrow/ :)

Reply:
@josevalim maybe like that?

![CleanShot 2023-09-14 at 15 27 09](https://github.com/livebook-dev/livebook.dev/assets/2719/97b214a1-2bc1-421b-8afe-80dcc53f80de)

Reply:
LGTM, your call.

Reply:
done ✅